Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -195,7 +198,7 @@ -(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
 - (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInteger)section
```

Comment:
Lots of repeated stuff like this. Is there a better way to do this where we always display the filtered events? Default with no search means "empty" filter which lets everything through and is equal to the sorted events. The sorted events could be stored in the background and we always present the "filtered" array to viewers.